templates/zerver/integrations.html

```diff
@@ -958,6 +958,11 @@
           <pre><code>HUBOT_ZULIP_BOT=hubot-bot@example.com HUBOT_ZULIP_API_KEY=your_key bin/hubot --adapter zulip --name "myhubot"</code></pre>
 
         </li>
+
+        <li>To run Hubot with a self-hosted version of Zulip, use::
+          <pre><code>HUBOT_ZULIP_SITE={{ external_api_path_subdomain }} HUBOT_ZULIP_BOT=hubot-bot@example.com HUBOT_ZULIP_API_KEY=your_key bin/hubot -a zulip</code></pre>
+
+        </li>
```

Comment:
This should just change the above block to set this parameter; the previous version doesn't work anymore with any server.